pkg/render/intrusion_detection.go

```diff
@@ -60,14 +61,18 @@ const (
 	IntrusionDetectionControllerPolicyName = networkpolicy.TigeraComponentPolicyPrefix + IntrusionDetectionControllerName
 	IntrusionDetectionInstallerPolicyName  = networkpolicy.TigeraComponentPolicyPrefix + "intrusion-detection-elastic"
 
-	ADJobPodTemplateBaseName      = "tigera.io.detectors"
-	adDetectorPrefixName          = "tigera.io.detector."
-	adDetectorName                = "anomaly-detectors"
-	ADDetectorPolicyName          = networkpolicy.TigeraComponentPolicyPrefix + adDetectorName
-	adDetectionJobsDefaultPeriod  = 15 * time.Minute
-	ADResourceGroup               = "detectors.tigera.io"
-	ADDetectorsModelResourceName  = "models"
-	ADLogTypeMetaDataResourceName = "metadata"
+	DefaultADStorageClassName              = "tigera-anomaly-detection"
+	DefaultAnomalyDetectionPVRequestSizeGi = 10
```

Comment:
Maybe you can simply change it to "10Gi", because all 4 times it is used, you string format to get that value.